modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring images in OCI format
+
+You can use the oc-mirror plug-in to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can then copy local OCI images to your target mirror registry.
+
+:FeatureName: Using the oc-mirror plug-in to mirror Operator images in OCI format
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+
+.Procedure
+
+. Create the image set configuration file to retrieve the catalogs and images that you require.
++
+.Example image set configuration file for copying to disk
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+ operators:
+ - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+   packages:
+   - name: aws-load-balancer-operator
+----
++
+[NOTE]
+====
+When using the OCI feature, the `storageConfig` setting is ignored in favor of the location passed in to the `oc mirror` command.
+====
++
+// TODO: I removed "helm" from these examples, that okay?
```

Comment:
@sherine-k @lmzuccarelli Since the `helm` field was empty, I removed them. Was that okay to do?

Reply:
@bergerhoffer - Maybe we can say that for the oci-feature we don't support the `heml` option in Tech Preview

Reply:
@lmzuccarelli I know we don't support the storageConfig. Is helm the only other field that isn't supported, or are there others that aren't supported? (Or really, is mirror.operators the only field that IS supported?)

Reply:
@bergerhoffer - So for the Tech Preview we only support "operators.catalog" fields (using the oci-feature flags).

For GA we are addressing "release" i.e "platform" and "additionalImages" fields,  we will also look at  the "storageConfig" options, but for now  "helm" is out of scope for GA

Reply:
Okay thanks - so instead of listing all of the other things we don't support for TP, I will work in that it is only operators.catalog that is supported.

Reply:
@lmzuccarelli I worked it in here: https://github.com/openshift/openshift-docs/pull/53150/files#diff-a7d9dab5e56bbeb4b2788c009eab5f2dcc4ee74e6a81b8b35a7d8bf97c2259b4R36-R41

Let me know if that works!